examples/RoleAssignment/RoleAssignment_Example.md

```diff
@@ -0,0 +1,50 @@
+# How To Run RoleAssignment.Tests.ps1
+
+`RoleAssignment.Tests.ps1` contains examples of using the `Confirm-AzBPRoleAssignment` cmdlet.
+
+## Pre-Requisites
+
+- Follow the [setup instructions](../README.md)
+
+## Steps
+
+1. Navigate to RoleAssignment directory:
+
+   ```Powershell
+   cd examples\RoleAssignment\
+   ```
+
+1. Deploy the Role Assignment to your subscription:
+  You will need Owner permission or higher
+
+   ```Powershell
+    New-AzSubscriptionDeployment -TemplateFile ".\roleAssignment.bicep" `
+    -Location "WestUS3"
+   ```
+
+1. When prompted in your terminal, set the `svcPrincipalObjectId` parameter with your service principal's object ID.
+
+1. Update `RoleAssignment.Tests.ps1` variables to point to your expected resources:
+
```

Comment:
Do you think it's necessary to call out the lack of resource group and location in the `.md` file?

Reply:
nice! I called that out in the file